test/controllers/api/v2/host_subscriptions_controller_test.rb

```diff
@@ -220,7 +222,7 @@ def test_content_override_bulk
     def test_content_override_accepts_string_values
       content_label = "some-content"
       value = "1"
-      assert_sync_task(::Actions::Katello::Host::UpdateContentOverrides) do |host, overrides|
+      assert_sync_task(::Actions::Katello::Host::UpdateContentOverrides) do |host, overrides, _|
```

Comment:
yeah '_' is the convention if you are not planning to use that variable. Else you will get an error from rubcop complaining "unused" variable